projects/packages/search/src/instant-search/components/overlay.scss

```diff
@@ -44,7 +44,33 @@ $overlay-horizontal-padding-lg: 3em;
 		padding: $overlay-vertical-padding-lg $overlay-horizontal-padding-lg;
 	}
 
-	h1, h2, h3, h4, h5, h6 {
+	button {
+		background-color: inherit !important;
```

Comment:
Just noticed we do already have a mixin called `@mixin remove-button-styling` that we could apply here instead (similar to the header styling below).

Reply:
nice spotting. I spent some with this mixin, but didn't have much luck with some of the elements. I suspect it's still being overridden by the twenty-twenty-one theme styles.